### PR TITLE
style: Fixed last commit details getting partially hidden

### LINKED ISSUE #4042: 🐛 Bug Report: Client - Last Commit panel is partially covered by the footer
### What happened?

The Last Commit panel is partially covered by the footer 

image

### What you expected to happen

The last commit panel should not be covered by the footer 

### How to reproduce

This only happens on a smaller screen when the height is not big enough to include the empty state image (of the box). 
To fix that, we should use a media-query to limit the size of the empty state image on smaller screens (based on the height) or we can just use an image width and height of 120px instead of 150px 

### Amplication version

1.0.2

### Environment

_No response_

### Are you willing to submit PR?

_No response_

### PR DESCRIPTION
**Pull request by TheLearneer**
Fixes #4042 

## PR Details

Fixed the styling issue where the last commit details were getting partially hidden

## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/VersionControl/PendingChanges.scss [hunk 1]
```diff
@@ -77,8 +77,9 @@ $resource-name-max-width: 180px;
     justify-content: flex-start;
     align-items: center;
 
-    img {
-      height: $empty-state-image-size;
+    .svg-theme-image {
+      width: 120px;
```
Comment (abrl91) [inline packages/amplication-client/src/VersionControl/PendingChanges.scss:81]: you have a variable at the top of this file that its value is 120px
`$empty-state-image-size: 120px;`
please use it.

Why did you add the width?

Comment (TheLearneer) [reply packages/amplication-client/src/VersionControl/PendingChanges.scss]: My bad, fixed the issue and used the variable

**Comment by abrl91** [comment]
@TheLearneer 
I saw Yuval's description on the issue:

_"This only happens on a smaller screen when the height is not big enough to include the empty state image (of the box).
To fix that, we should use a media query to limit the size of the empty state image on smaller screens (based on the height), or we can just use an image width and height of 120px instead of 150px"_

As the image height was already 120px, can you please apply the change you have made to the `textarea` height with a media query? Meaning, that only on small screens the `textarea` height would be smaller?

**Comment by TheLearneer** [comment]
Hi @abrl91 Updated the code to include media query for text area

**Comment by abrl91** [comment]
run branch

**Review by abrl91 — APPROVED** [review]
